server/routes/caseNotes/caseNotesView.ts

```diff
@@ -0,0 +1,32 @@
+import { TableArgs } from '../../utils/govukFrontendTypes'
+import CaseNotesPresenter from './caseNotesPresenter'
+import ViewUtils from '../../utils/viewUtils'
+
+export default class CaseNotesView {
+  constructor(private presenter: CaseNotesPresenter) {}
+
+  private get tableArgs(): TableArgs {
+    return {
+      head: [
+        { text: 'Details', classes: 'govuk-!-width-one-quarter' },
+        { text: 'Case notes', classes: 'govuk-!-width-three-quarters' },
+      ],
+      rows: this.presenter.tableRows.map(row => {
+        return [
+          { html: `${row.sentAtDateTime}<br>${row.sentBy}` },
+          { html: `<b>${ViewUtils.escape(row.subject)}</b><br><br>${ViewUtils.escape(row.body)}` },
```

Comment:
please can we use govuk styles here?

Reply:
it could also be worth seeing how the first cell looks with two paragraphs instead of a line break.